.pre-commit-config.yaml

```diff
@@ -6,10 +6,6 @@ repos:
     hooks:
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
--   repo: https://github.com/pre-commit/mirrors-autopep8
```

Comment:
Why did this change?

Reply:
This comment has more context: https://github.com/Shopify/shopify_python_api/pull/479#discussion_r591842577